packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
+  let value = path.value();
+
+  if (value === true) {
+    return dasherize(className.value());
```

Comment:
the last arg is the name of the ref passed in, the ref is the className unless it is a boolean.